training-maquetado/scss/_titles.scss

```diff
@@ -0,0 +1,18 @@
+@import "_styles";
+
+$title-decoration-height: 90px;
+
+$title-padding-left: 51px;
+
+.title-box {
+  align-items: center;
+  background: url(assets/triangle.svg) left center no-repeat;
```

Comment:
in this case i would add the image with the `::before` selector
https://developer.mozilla.org/es/docs/Web/CSS/::before